src/Databases/DatabaseAtomic.cpp

```diff
@@ -518,11 +558,16 @@ void DatabaseAtomic::tryCreateSymlink(const String & table_name, const String &
     }
 }
 
+String DatabaseAtomic::GetPathSymlink(const String & table_name) const
```

Comment:
```suggestion
String DatabaseAtomic::getPathSymlink(const String & table_name) const
```

https://clickhouse.com/docs/en/development/style